File: documentation-examples-scala/src/scala/dev/typr/foundationssc/docs/db2/SpecialTypes.scala
package dev.typr.foundationssc.docs.db2
import dev.typr.foundationssc.*
import dev.typr.foundationssc.data.*



@SuppressWarnings(Array("unused"))
object SpecialTypes:
  //start
  val xmlType: Db2Type[Xml] = Db2Types.xml
  val rowidType: Db2Type[Array[Byte]] = Db2Types.rowid
  //stop

File: documentation-examples-kotlin/src/kotlin/dev/typr/foundationskt/docs/db2/SpecialTypes.kt
package dev.typr.foundationskt.docs.db2

import dev.typr.foundationskt.*
import dev.typr.foundationskt.data.*

@Suppress("unused")
class SpecialTypes {
    //start
    val xmlType: Db2Type<Xml> = Db2Types.xml
    val rowidType: Db2Type<ByteArray> = Db2Types.rowid
    //stop
}
